spikefinder/main.py

```diff
@@ -34,13 +41,150 @@ def _corr(x, y):
     return corrcoef(x, y)[0,1]
 
 def _info(x, y):
-    pass
+    loglik, info = _infolik(x,y)
+    return info       # this is absolute info gain, we report mean(info)/mean(entropy) across all cells in the paper
 
 def _rank(x, y):
-    pass
+    
+    return spearmanr(x,y,nan_policy='raise').correlation        
 
 def _auc(x, y):
-    pass
+     pass
 
 def _loglik(x, y):
-    pass
\ No newline at end of file
+    
+    loglik, info = _infolik(x,y)
+    return loglik
+
+def _downsample(signal, factor):
```

Comment:
I usually prefer the `numpy` docstring style, described [here](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#sections), I can easily fix these ones, but FYI for the next one!